model_compression_toolkit/core/common/quantization/set_node_quantization_config.py

```diff
@@ -79,6 +79,68 @@ def set_quantization_configuration_to_graph(graph: Graph,
     return graph
 
 
+def filter_node_qco_by_graph(node: BaseNode, tpc: TargetPlatformCapabilities,
+                             graph: Graph, node_qc_options: QuantizationConfigOptions
+                             ) -> Tuple[OpQuantizationConfig, List[OpQuantizationConfig]]:
```

Comment:
Please start a new line for each argument, and detail in the comments what it returns.
Also, is it possible to split this function insto smaller functions?